ocfweb/docs/docs/services/account/content-removal.md

```diff
@@ -0,0 +1,63 @@
+[[!meta title="Content Removal Policy"]]
+
+
+## Introduction
+
+Most of the content on [www.ocf.berkeley.edu](https://www.ocf.berkeley.edu)
+is created and owned by OCF members, not the OCF itself. This user-generated
+content is not endorsed or supported by the OCF nor is the OCF responsible
+for content posted by OCF members.
+
+The OCF maintains a strict policy not to remove or modify data
+hosted by members unless legally compelled to do so, or if such
+data is in violation of existing OCF
+[[account policies|doc services/account/account-policies]].
+
+This means the OCF will not change or remove any text, pictures,
+information, or other data hosted by an OCF member. We do however
```

Comment:
This sentence sounds like it contradicts the previous paragraph wrt forced removals